src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -392,7 +419,23 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
               @value-changed=${this._areaPicked}
             ></ha-area-picker>`
           : ""}
-
+        ${this._cameraPrefs
+          ? html`
+              <ha-settings-row>
+                <span slot="heading">Preload camera stream</span>
+                <span slot="description"
+                  >This keeps the camera stream open in the background so it
+                  shows quicker. Warning! This can slow down your instance
+                  drastically.</span
```

Comment:
For in localize:
```suggestion
                  >This keeps the camera stream open in the background so it
                  shows quicker. Warning! This is CPU/network intensive.</span
```

Reply:
We want to move from CPU though. Hardware /Network ?

Reply:
I agree on `Hardware`

Reply:
I don't understand the comments. You're saying you want it to say "This is hardware intensive"?

Reply:
CPU is a bit technical in my opinion. Instead of hardware, maybe device?